src/Maestro/Client/src/MaestroApi.cs

```diff
@@ -80,31 +92,32 @@ public partial class MaestroApi
             // 1. BAR or Entra token that can directly be used to authenticate against Maestro
```

Comment:
I am wondering whether this whole method could also not be part of the `Common` library. I think the PCS version is a bit different but probably there would not be any harm in unifying this too.

Reply:
But also not a strong opinion

Reply:
Yeah, thought about it too but maybe where would be cases where we didn't want to support certain auth flows or would need to alter their order for a given service? Not sure

Reply:
I meant - keep both - the various public methods for specific credentials as well as the master method that you can pass everything to and it will prioritize

Reply:
Done